packages/pegasus/src/install-on-chain.js

```diff
@@ -0,0 +1,45 @@
+// @ts-check
+import { E } from '@agoric/eventual-send';
+
+export const CONTRACT_NAME = 'Pegasus';
+
+// TODO: TECHDEBT: ambient types don't work here.
+/* @param { BootstrapPowers } powers */
+export async function installOnChain({
+  consume: {
+    agoricNames,
+    board,
+    nameAdmins,
+    namesByAddress,
+    pegasusBundle: bundleP,
+    zoe,
+  },
+}) {
+  // Fetch the nameAdmins we need.
+  const [installAdmin, instanceAdmin] = await Promise.all(
```

Comment:
I'm struggling to understand NameHubs and NameAdmins. The annotations on their source code only describe the type signatures of the APIs and don't give any hint about their purpose or how they contribute to POLA (which I infer is the point.)

It appears that they're being used here to segregate groups of powers. Is there a description of what the various nameAdmins represent?